spores-pickling/src/test/scala/scala/spores/run/pickling/Issue15.scala

```diff
@@ -0,0 +1,47 @@
+/*                     __                                               *\
+ **     ________ ___   / /  ___     Scala API                            **
+ **    / __/ __// _ | / /  / _ |    (c) 2002-2013, LAMP/EPFL             **
+ **  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
+ ** /____/\___/_/ |_/____/_/ | |                                         **
+ **                          |/                                          **
+\*                                                                      */
+
+package scala.spores
+package run
+package pickling
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+import scala.pickling._
+import Defaults._
+import json._
+import static._
+
+import SporePicklers._
+
+@RunWith(classOf[JUnit4])
+class Issue15Spec {
+
+  @Test
+  def test(): Unit = {
+    val s = spore {
+      val pre = "hello"
+      l: List[Int] => l.map(x => s"$pre $x")
+    }
+
+    val d = s.pickle
+    System.out.println(d.value.toString)
```

Comment:
You sure we want to remove it? By default it's not executed, only when tests fail. I find it very useful... when tests are green, the test report is clean, otherwise it's dirty.